maintainers/maintainer-list.nix

```diff
@@ -8568,6 +8568,12 @@
     github = "polygon";
     githubId = 51489;
   };
+  polykernel = {
+    email = "81340136+polykernel@users.noreply.github.com"
```

Comment:
```suggestion
    email = "81340136+polykernel@users.noreply.github.com";
```

Reply:
thanks for catching this careless mistake.